### PR TITLE
[s3] do reload s3 static config

### PR DESCRIPTION
**Pull request by kmlebedev**
# What problem are we solving?

Updating the configuration with `kill -s HUP`

# How are we solving the problem?

add kill signal handler 

# How is the PR tested?

# Checks
- [ ] I have added unit tests if possible.
- [ ] I will add related wiki document changes and link to this PR after merging.

### CODE REVIEW

File: weed/s3api/s3api_server.go [hunk 1]
```diff
@@ -61,6 +62,11 @@ func NewS3ApiServer(router *mux.Router, option *S3ApiServerOption) (s3ApiServer
 		filerGuard:     security.NewGuard([]string{}, signingKey, expiresAfterSec, readSigningKey, readExpiresAfterSec),
 		cb:             NewCircuitBreaker(option),
 	}
+	if option.Config != "" {
+		grace.OnReload(func() {
+			_ = s3ApiServer.iam.loadS3ApiConfigurationFromFile(option.Config)
```
Comment (chrislusf) [inline weed/s3api/s3api_server.go]: add some logging here?

And some wiki changes after the merge.

**Comment by kmlebedev** [comment]
Done

File: weed/s3api/s3api_server.go [hunk 2]
```diff
@@ -67,6 +67,8 @@ func NewS3ApiServer(router *mux.Router, option *S3ApiServerOption) (s3ApiServer
 		grace.OnReload(func() {
 			if err := s3ApiServer.iam.loadS3ApiConfigurationFromFile(option.Config); err != nil {
 				glog.Errorf("fail to load config file %s: %v", option.Config, err)
+			} else {
+				glog.V(0).Infof("success loaded %s identities from config file %s", len(s3ApiServer.iam.identities), option.Config)
```
Comment (chrislusf) [inline weed/s3api/s3api_server.go]:
```suggestion
				glog.V(0).Infof("Loaded %s identities from config file %s", len(s3ApiServer.iam.identities), option.Config)
```

**Comment by kmlebedev** [comment]
Add wiki changes 
https://github.com/seaweedfs/seaweedfs/wiki/Amazon-S3-API/_compare/ab1de599451ba77bab0c11ba4720c53e269d7198...3379e144b6408a2e5579bca353adcae962601177

**Pull request merged**